test/lib/commands/audit.js

```diff
@@ -1198,6 +1198,14 @@ t.test('audit signatures', async t => {
       npm.exec('audit', ['signatures']),
       /found no dependencies to audit that where installed from a supported registry/
     )
+
+    // Some registries return 400 instead, even though 404 would be more appropriate
+    registry.nock.get('/-/npm/v1/keys').reply(400)
+
+    await t.rejects(
```

Comment:
`npm.exec` shouldn't be reused. there's a lot going on under the hood every time npm is instantiated and ran, this needs a separate test

Reply:
I've updated it to be a separate test.